### PR TITLE
Add score calculation script

### PR DESCRIPTION
**Pull request by udpatil**
Currently we support only create_validator mission scoring, I'll add other mission validating / scoring shortly in future PRs.

TLDR flow of script:

1. obtain TSV of google form responses
2. script parses the TSV into FormEntries (which is a row from the google form)
3. Form Entries are then processed to extract tx hashes out of the proof of completion
4. We then query sei-node-0 grpc endpoint to get back TX info for the tx hash
5. We validate the response to identify what mission it correlates to
6. We track the missions completed in the score singleton (this also handles deduplicating missions for each discord handle AND prevents reusing tx hashes as proof of completion by multiple people - TODO still need to test this last part better)
7. After validating all missions, generate a score dictionary keyed by discord handle and write to an output CSV file.

### CODE REVIEW

**Review by LCyson — APPROVED** [review]
can we not merge to the public repo, but to the sei-infra repo instead
